frontend/src/metabase/parameters/components/FormattedParameterValue/FormattedParameterValue.tsx

```diff
@@ -33,25 +39,42 @@ function FormattedParameterValue({
 
   const label = getLabel(displayValue);
 
-  if (
-    isFieldFilterUiParameter(parameter) &&
-    hasFields(parameter) &&
-    !isDateParameter(parameter)
-  ) {
+  const renderContent = () => {
```

Comment:
Nit: you could make this a separate component like `FormattedParameterValueInner` and pass `parameter`, `value`, `label` as props.